_posts/2022-04-06-T2T-human-genome.md

```diff
@@ -0,0 +1,30 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'T2T-CHM13v2.0 available in Galaxy'
+---
+
+Despite the first version of the human genome generated as a result of the Telomere-to-Telomere (T2T) Consortium efforts dates back to 2020, it has taken two years of additional work to achieve something that the scientific community was eagerly awaiting: the first complete, gap-free sequence for all 24 human chromosomes.
```

Comment:
```suggestion
While the first version of an improved human genome resulting from the Telomere-to-Telomere (T2T) Consortium efforts got reported in 2020, it has taken two years of additional work to achieve something that the scientific community has been eagerly awaiting since: the first complete, gap-free sequence for all 24 human chromosomes (including Y).

```

Reply:
Do you have a citation for their version 1?

Reply:
I took as reference the first entry in NCBI (the version 0.7 https://www.ncbi.nlm.nih.gov/assembly/GCA_009914755.1/)